Speech Async sample not working

[daharon-macpro:captionasync daharon$ go run captionasync.go 
captionasync.go:17:2: cannot find package "github.com/gogo/protobuf/proto" in any of:
	/usr/local/go/src/github.com/gogo/protobuf/proto (from $GOROOT)
	/Users/daharon/go/src/github.com/gogo/protobuf/proto (from $GOPATH)

--- a/speech/captionasync/captionasync.go
+++ b/speech/captionasync/captionasync.go
@@ -14,7 +14,7 @@ import (
 	"os"
 	"time"
 
-	"github.com/gogo/protobuf/proto"
+	"github.com/golang/protobuf/proto"
 
 	"golang.org/x/net/context"
 

